index.jsx

```diff
@@ -137,11 +137,19 @@ module.exports = function Dimensions ({
         this.getWindow().removeEventListener('resize', this.onResize)
       }
 
+      getWrappedInstance = () => this.refs.wrappedInstance;
```

Comment:
Oh yeah sorry I think it would work as a regular non-bound function. I tend to bind by default so I don't have to think about it but you are right